pkgs/dartpad_ui/lib/local_storage/_web.dart

```diff
@@ -6,15 +6,15 @@ import 'package:dartpad_shared/model.dart';
 import 'package:web/web.dart' as web;
 
 import '../utils.dart';
-import 'interface.dart';
+import '_stub.dart' as stub;
 
 const _userInputKey = 'user_input_';
 const _userKeybindingKey = 'user_keybinding_';
 const _lastCreateCodePromptKey = 'last_create_code_prompt_';
 const _lastUpdateCodePromptKey = 'last_update_code_prompt_';
 const _lastCreateCodeAppTypeKey = 'last_create_code_app_type_';
 
-class LocalStorageImpl extends LocalStorageInterface {
```

Comment:
Why not keep the same interface with two separate implementations?

If there's no interface, we could consider renaming this class to `LocalStorage` or `DartPadLocalStorage`

Reply:
With interface it will be extra file and extra element, that seems to be overdesign.

Renamed. Thanks for suggestion.